docs/scripts_md/MakeArchiveLocationRelative.md

```diff
@@ -0,0 +1,53 @@
+# NAME
+
+MakeArchiveLocationRelative.pl -- Removes the root directory from the
+`ArchiveLocation` field in the `tarchive` table to make the path to the
+tarchive relative.
+
+# SYNOPSIS
+
+perl MakeArchiveLocationRelative.pl `[options]`
+
+Available option is:
+
+\-profile: name of the config file in ../dicom-archive/.loris\_mri
```

Comment:
I think this path is no longer valid if the file is moved to the tools/ directory; don't you think?

Reply:
Since I am moving the script in the tools directory based on your suggestion below, it becomes valid now so I leave it like this.